src/locales/tr.json

```diff
@@ -100,6 +100,9 @@
     }
   },
   "games": {
+    "quizGame": {
+      "title": "sınav oyunu"
```

Comment:
You can capitalize first letters "Sınav Oyunu"

Reply:
PTAL